dhis-2/dhis-web/dhis-web-api/src/main/java/org/hisp/dhis/webapi/controller/event/EventController.java

```diff
@@ -67,11 +65,7 @@
 import org.hisp.dhis.event.EventStatus;
 import org.hisp.dhis.fieldfilter.FieldFilterParams;
 import org.hisp.dhis.fieldfilter.FieldFilterService;
-import org.hisp.dhis.fileresource.FileResource;
-import org.hisp.dhis.fileresource.FileResourceDomain;
-import org.hisp.dhis.fileresource.FileResourceService;
-import org.hisp.dhis.fileresource.FileResourceStorageStatus;
-import org.hisp.dhis.fileresource.ImageFileDimension;
+import org.hisp.dhis.fileresource.*;
```

Comment:
don't collapse imports

Reply:
I use the official style template. So if we do not want to collapse them, then we should change it there and let know the people to update.

But I have also turned off automatic optimization of imports on commit.